ci/test/build.py

```diff
@@ -29,6 +29,7 @@ def main() -> None:
     print("--- Acquiring mzbuild images")
     deps = repo.resolve_dependencies(image for image in repo if image.publish)
     deps.ensure()
+    mzbuild.publish_multiarch_images(f'devel-{git.rev_parse("HEAD")}', deps)
```

Comment:
Ah, shoot, this is not going to be so simple. `publish_multiarch_images` needs _two_ dependency sets: one for aarch64 and one for x86_64. It looks like we'll need a separate job in the build pipeline that waits on both the x86_64 and aarch64 build jobs, then constructs the two dep sets, and _then_ publishes the multiarch images.

Reply:
Hm, can't this be done incrementally, as two pushes? I felt like that might be possible...

Reply:
Anyway, additional build step it is (:

Reply:
> Hm, can't this be done incrementally, as two pushes? I felt like that might be possible...

Not easily, no! The manifest you push up to Docker Hub needs to have both images inside of it.